scripts/run_frontend_tests.sh

```diff
@@ -37,28 +37,60 @@ set -e
 source $(dirname $0)/setup.sh || exit 1
 source $(dirname $0)/setup_gae.sh || exit 1
 
-# Install third party dependencies
-# TODO(sll): Make this work with fewer third-party dependencies.
-bash scripts/install_third_party.sh
-
-# Ensure that generated JS and CSS files are in place before running the tests.
-echo ""
-echo "  Running build task with concatenation only "
-echo ""
+# Parse additional command line arguments.
```

Comment:
Done. This might slightly impact the performance of running the frontend tests before dependencies are installed (since it will also now install the protractor dependencies for both, ditto for end-to-end tests), but it's inconsequential.